.github/workflows/poc-test-environments.yml

```diff
@@ -30,4 +30,11 @@ jobs:
         with:
           node-version: lts/*
           cache: 'yarn'
-      # TODO: add yarn environments <category> pull incrementally
+      - name: Add Amplify CLI
+        run: yarn global add @aws-amplify/cli
+
+      - name: Pull down Geo AWS environments, on cache miss
```

Comment:
This is only for POC purposes to demonstrate that `yarn environments geo pull` works. See "status checks" below.